hack/update-protobuf.sh

```diff
@@ -6,6 +6,7 @@ PACKAGES=(
     github.com/sensu/sensu-go/internal/apis/meta
     github.com/sensu/sensu-go/internal/apis/rbac
     github.com/sensu/sensu-go/internal/apis/core
+    github.com/sensu/sensu-go/apis/meta/v1alpha1
```

Comment:
It might be a stupid question, but should this touch frozen APIs packages?
I would expect _any_ change to those frozen packages to _only_ come from modifications of the internal API + run of the `freeze-api` tool.

Reply:
Hmm. That is probably how it _should_ work. But we have some technical debt to clear out of the way first. (i.e., the hack script)